app/invocation/invocation.css

```diff
@@ -1030,6 +1032,36 @@
   color: #aaa;
 }
 
+.action-list .expected-output {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+
+  .icon {
+    opacity: 0.7;
+  }
+
+  .expected-output-label {
+    color: #444;
+  }
+}
+
+.action-list .missing-output {
+  display: flex;
+  align-items: center;
+
+  .icon {
+    height: 18px;
+    width: 18px;
+    opacity: 0.7;
+    stroke: red;
```

Comment:
nit: add the `icon-red` class to the icon instead, to get the material red color that we use elsewhere in the UI

Reply:
err it's actually just `red` not `icon-red`. so `className="icon red"`